src/middleware/sessionStore.js

```diff
@@ -0,0 +1,56 @@
+import {connectionAPI, connectionDefault} from '../config'
+import {Schema} from 'mongoose'
+
+/**
+ * Session Store
+ *
+ * To be able to store the session in a Mongo collection instead of just saving it in memory
+ * 
+ */
+
+const SessionSchema = new Schema({
+  _id: String,
+  data: Object,
+  updatedAt: {
+    default: new Date(),
+    type: Date
+  }
+});
+
+class MongooseStore {
+  constructor() {
+    this.sessionModel = connectionDefault.model('Session', SessionSchema)
+    this.session = connectionAPI.model('Session', SessionSchema)
```

Comment:
Why do we need two models ?

Reply:
I used the same pattern as the other models.

there is different connection types in the code one with options and one without, 
so I stuck to the same way the other models were defined

Reply:
So one is used for UI and the other for the API, we don't need the session for the API. right ?

Reply:
cool done